### PR TITLE
ChIPseeker

### PR DESCRIPTION
**Pull request by gyear**
For INC1209490 - `ChIPseeker-1.28.3-foss-2021a-R-4.1.0.eb`

* [x] Assigned to reviewers (usually everyone in apps team)

Default:
* [x] EL8-cascadelake
* [x] EL8-haswell
* [x] EL8-icelake

### CODE REVIEW

File: easybuild/easyconfigs/c/ChIPseeker/ChIPseeker-1.30.3-foss-2021a-R-4.1.0.eb [hunk 1]
```diff
@@ -0,0 +1,93 @@
+# This easyconfig was created by the BEAR Software team at the University of Birmingham.
+easyblock = 'Bundle'
+
+name = 'ChIPseeker'
+version = '1.30.3'
```
Comment (branfosj) [inline easybuild/easyconfigs/c/ChIPseeker/ChIPseeker-1.30.3-foss-2021a-R-4.1.0.eb]: https://bioconductor.org/packages/3.13/bioc/html/ChIPseeker.html for the version of ChIPseeker from Bioconductor 3.13.

So this should be `version = '1.28.3'`

File: easybuild/easyconfigs/c/ChIPseeker/ChIPseeker-1.30.3-foss-2021a-R-4.1.0.eb [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+dependencies = [
+    ('R', '4.1.0'),
+    ('R-bundle-Bioconductor', '3.13', '-R-%(rver)s'),
+]
+
+exts_defaultclass = 'RPackage'
+exts_filter = ("R -q --no-save", "library(%(ext_name)s)")
+exts_default_options = {
+    'source_urls': [
+        'https://bioconductor.org/packages/release/bioc/src/contrib/',  # my addition
```
Comment (branfosj) [inline easybuild/easyconfigs/c/ChIPseeker/ChIPseeker-1.30.3-foss-2021a-R-4.1.0.eb]:
```suggestion
```

That will get packages from Bioconductor 3.14, which can cause odd issues. Use https://bioconductor.org/packages/3.13/BiocViews.html#___Software as a starting point.

I've suggested the ones below that I think need to change due to this - the others are from CRAN. Checksums will need updating and then test build this to check that the package dependencies are fine.

File: easybuild/easyconfigs/c/ChIPseeker/ChIPseeker-1.30.3-foss-2021a-R-4.1.0.eb [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+        'https://cran.r-project.org/src/contrib/Archive/%(name)s',  # package archive
+        'https://cran.r-project.org/src/contrib/',  # current version of packages
+        'https://cran.freestatistics.org/src/contrib',  # mirror alternative for current packages
+    ],
+    'source_tmpl': '%(name)s_%(version)s.tar.gz',
+}
+
+# Order is important!
+exts_list = [
+    ('GOSemSim', '2.20.0', {
```
Comment (branfosj) [inline easybuild/easyconfigs/c/ChIPseeker/ChIPseeker-1.30.3-foss-2021a-R-4.1.0.eb]:
```suggestion
    ('GOSemSim', '2.18.1', {
```

File: easybuild/easyconfigs/c/ChIPseeker/ChIPseeker-1.30.3-foss-2021a-R-4.1.0.eb [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+        'checksums': ['4b81261ef5cddbadc1a3cf01025a0e179ae07ac06956bf8a7807aa3767321197'],
+    }),
+    ('DO.db', '2.9', {
+        'checksums': ['762bcb9b5188274fd81d82f785cf2846a5acc61fad55e2ff8ec1502282c27881'],
+    }),
+    ('DOSE', '3.20.1', {
```
Comment (branfosj) [inline easybuild/easyconfigs/c/ChIPseeker/ChIPseeker-1.30.3-foss-2021a-R-4.1.0.eb]:
```suggestion
    ('DOSE', '3.18.3', {
```

File: easybuild/easyconfigs/c/ChIPseeker/ChIPseeker-1.30.3-foss-2021a-R-4.1.0.eb [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+    ('scatterpie', '0.1.7', {
+        'checksums': ['3f7807519cfe135066ca79c8d8a09b59da9aa6d8aaee5e9aff40cca3d0bebade'],
+    }),
+    ('shadowtext', '0.1.1', {
+        'checksums': ['eb06581d7ed06c963eee47548932688fd48eba70b3ebd2a7b41a6501d6e00006'],
+    }),
+    ('tidytree', '0.3.7', {
+        'checksums': ['7816f2d48ec94ca0c1bef15ec3d536adf44a969ea3c3cfc203ceebe16808e4f2'],
+    }),
+    ('treeio', '1.18.1', {
```
Comment (branfosj) [inline easybuild/easyconfigs/c/ChIPseeker/ChIPseeker-1.30.3-foss-2021a-R-4.1.0.eb]:
```suggestion
    ('treeio', '1.16.2', {
```

File: easybuild/easyconfigs/c/ChIPseeker/ChIPseeker-1.30.3-foss-2021a-R-4.1.0.eb [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+        'checksums': ['5805393b524906d801602ddac126319ac7567de9afd80bfbfaa76ce6379428a6'],
+    }),
+    ('ggtree', '3.2.1', {
```
Comment (branfosj) [inline easybuild/easyconfigs/c/ChIPseeker/ChIPseeker-1.30.3-foss-2021a-R-4.1.0.eb]:
```suggestion
    ('ggtree', '3.0.4', {
```

File: easybuild/easyconfigs/c/ChIPseeker/ChIPseeker-1.30.3-foss-2021a-R-4.1.0.eb [hunk 7]
```diff
@@ -0,0 +1,93 @@
...
+        'checksums': ['6803a1dd37c6fd86fa1ec7ec6a62c8423676b5f045c45da1b47f127fdb4b6362'],
+    }),
+    ('enrichplot', '1.14.1', {
```
Comment (branfosj) [inline easybuild/easyconfigs/c/ChIPseeker/ChIPseeker-1.30.3-foss-2021a-R-4.1.0.eb]:
```suggestion
    ('enrichplot', '1.12.3', {
```

**Review by branfosj — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/c/ChIPseeker/ChIPseeker-1.30.3-foss-2021a-R-4.1.0.eb (hunk 1 above)
Comment (branfosj) [reply easybuild/easyconfigs/c/ChIPseeker/ChIPseeker-1.30.3-foss-2021a-R-4.1.0.eb]: Update comment above, which will hopefully make more sense

Comment (edmondac) [reply easybuild/easyconfigs/c/ChIPseeker/ChIPseeker-1.30.3-foss-2021a-R-4.1.0.eb]: This will need the filename changing too

**Review by branfosj — APPROVED** [review]

**Pull request merged**